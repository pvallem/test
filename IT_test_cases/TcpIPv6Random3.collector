bidir_flow_info {
  key {
    src_address: "J\321\014\265C\223\341C*\306_\343&W\315\034"
    dst_address: "\310\\\345\246\226DDm\t\377\362pK\272\222\317"
    src_port: 26843
    dst_port: 24093
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: IPV6
  }
  flow_info {
    start_time: 1388563200000000
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 0
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000000
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "J\321\014\265C\223\341C*\306_\343&W\315\034"
    dst_address: "\243\ry\251\316\245\245b\250\207\220\377\376\353C\201"
    src_port: 8147
    dst_port: 32496
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200003000
    key_type: IPV6
  }
  flow_info {
    start_time: 1388563200003000
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200003000
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\213\215\021\202\327\355\321\036WE\232\030\232\033\256\375"
    dst_address: "\225\000\312*\216\025i2\022\371\376\212Ow\266\003"
    src_port: 8049
    dst_port: 58444
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200002000
    key_type: IPV6
  }
  reverse_flow_info {
    start_time: 1388563200002000
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 20
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200002000
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\217\246\'\305\272\273\300\363AlYQ\304w\333\215"
    dst_address: "\356\343\214H\322*x{\256j(3\367G\326("
    src_port: 25018
    dst_port: 1716
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200001000
    key_type: IPV6
  }
  reverse_flow_info {
    start_time: 1388563200001000
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200001000
    sensor_id: "127.0.0.1:5675"
  }
}
