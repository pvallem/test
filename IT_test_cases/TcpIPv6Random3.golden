flow_info {
  key {
    src_address: "f\217\013\347nj\250\253\252d]\276I\263\274\267"
    dst_address: "\307\334\327\251\211\302\006\365\305L0\240\363\005\241$"
    src_port: 1070
    dst_port: 9802
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: IPV6
  }
  start_time: 1388563200000000
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 0
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000000
}
flow_info {
  key {
    src_address: "\262\256\2503\262\002\322\273e\305\3474^6`\004"
    dst_address: "\341\2030\234\027\265\'\024\376\326\200<>\005\367{"
    src_port: 5060
    dst_port: 39060
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200057961
    key_type: IPV6
  }
  start_time: 1388563200057961
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200057961
}
flow_info {
  key {
    src_address: "p\202q\266j\035\024\242N\362}{!r\204\251"
    dst_address: "\330\272!\320\362p\264\006\020w3\246\177\212\206N"
    src_port: 61501
    dst_port: 2463
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200115922
    key_type: IPV6
  }
  start_time: 1388563200115922
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 20
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200115922
}
flow_info {
  key {
    src_address: "f\217\013\347nj\250\253\252d]\276I\263\274\267"
    dst_address: "\217\356\231\256\212\211=GU\324\245\261$\235g/"
    src_port: 40894
    dst_port: 11217
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200173883
    key_type: IPV6
  }
  start_time: 1388563200173883
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200173883
}
flow_info {
  key {
    src_address: "\262\256\2503\262\002\322\273e\305\3474^6`\004"
    dst_address: "\262\256\2503\262\002\322\273e\305\3474^6`\004"
    src_port: 51344
    dst_port: 61158
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200231844
    key_type: IPV6
  }
  start_time: 1388563200231844
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 40
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200231844
}
flow_info {
  key {
    src_address: "p\202q\266j\035\024\242N\362}{!r\204\251"
    dst_address: "\307\334\327\251\211\302\006\365\305L0\240\363\005\241$"
    src_port: 17217
    dst_port: 37778
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200289805
    key_type: IPV6
  }
  start_time: 1388563200289805
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200289805
}
flow_info {
  key {
    src_address: "f\217\013\347nj\250\253\252d]\276I\263\274\267"
    dst_address: "\341\2030\234\027\265\'\024\376\326\200<>\005\367{"
    src_port: 39353
    dst_port: 4921
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200347766
    key_type: IPV6
  }
  start_time: 1388563200347766
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 60
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200347766
}
flow_info {
  key {
    src_address: "\262\256\2503\262\002\322\273e\305\3474^6`\004"
    dst_address: "\330\272!\320\362p\264\006\020w3\246\177\212\206N"
    src_port: 1070
    dst_port: 23104
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200405727
    key_type: IPV6
  }
  start_time: 1388563200405727
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200405727
}
flow_info {
  key {
    src_address: "p\202q\266j\035\024\242N\362}{!r\204\251"
    dst_address: "\217\356\231\256\212\211=GU\324\245\261$\235g/"
    src_port: 5060
    dst_port: 63752
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200463688
    key_type: IPV6
  }
  start_time: 1388563200463688
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 80
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200463688
}
flow_info {
  key {
    src_address: "f\217\013\347nj\250\253\252d]\276I\263\274\267"
    dst_address: "\262\256\2503\262\002\322\273e\305\3474^6`\004"
    src_port: 61501
    dst_port: 33279
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200464688
    key_type: IPV6
  }
  start_time: 1388563200464688
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200464688
}
flow_info {
  key {
    src_address: "\262\256\2503\262\002\322\273e\305\3474^6`\004"
    dst_address: "\307\334\327\251\211\302\006\365\305L0\240\363\005\241$"
    src_port: 40894
    dst_port: 9795
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200522649
    key_type: IPV6
  }
  start_time: 1388563200522649
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 100
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200522649
}
flow_info {
  key {
    src_address: "p\202q\266j\035\024\242N\362}{!r\204\251"
    dst_address: "\341\2030\234\027\265\'\024\376\326\200<>\005\367{"
    src_port: 51344
    dst_port: 9802
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200580610
    key_type: IPV6
  }
  start_time: 1388563200580610
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200580610
}
flow_info {
  key {
    src_address: "f\217\013\347nj\250\253\252d]\276I\263\274\267"
    dst_address: "\330\272!\320\362p\264\006\020w3\246\177\212\206N"
    src_port: 17217
    dst_port: 39060
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200638571
    key_type: IPV6
  }
  start_time: 1388563200638571
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 120
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200638571
}
flow_info {
  key {
    src_address: "\262\256\2503\262\002\322\273e\305\3474^6`\004"
    dst_address: "\217\356\231\256\212\211=GU\324\245\261$\235g/"
    src_port: 39353
    dst_port: 2463
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200696532
    key_type: IPV6
  }
  start_time: 1388563200696532
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200696532
}
flow_info {
  key {
    src_address: "p\202q\266j\035\024\242N\362}{!r\204\251"
    dst_address: "\262\256\2503\262\002\322\273e\305\3474^6`\004"
    src_port: 1070
    dst_port: 11217
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200754493
    key_type: IPV6
  }
  start_time: 1388563200754493
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 140
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200754493
}
flow_info {
  key {
    src_address: "f\217\013\347nj\250\253\252d]\276I\263\274\267"
    dst_address: "\307\334\327\251\211\302\006\365\305L0\240\363\005\241$"
    src_port: 5060
    dst_port: 61158
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200812454
    key_type: IPV6
  }
  start_time: 1388563200812454
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200812454
}
flow_info {
  key {
    src_address: "\262\256\2503\262\002\322\273e\305\3474^6`\004"
    dst_address: "\341\2030\234\027\265\'\024\376\326\200<>\005\367{"
    src_port: 61501
    dst_port: 37778
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200870415
    key_type: IPV6
  }
  start_time: 1388563200870415
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 160
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200870415
}
flow_info {
  key {
    src_address: "p\202q\266j\035\024\242N\362}{!r\204\251"
    dst_address: "\330\272!\320\362p\264\006\020w3\246\177\212\206N"
    src_port: 40894
    dst_port: 4921
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200928376
    key_type: IPV6
  }
  start_time: 1388563200928376
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200928376
}
flow_info {
  key {
    src_address: "f\217\013\347nj\250\253\252d]\276I\263\274\267"
    dst_address: "\217\356\231\256\212\211=GU\324\245\261$\235g/"
    src_port: 51344
    dst_port: 23104
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200929376
    key_type: IPV6
  }
  start_time: 1388563200929376
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 180
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200929376
}
flow_info {
  key {
    src_address: "\262\256\2503\262\002\322\273e\305\3474^6`\004"
    dst_address: "\262\256\2503\262\002\322\273e\305\3474^6`\004"
    src_port: 17217
    dst_port: 63752
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200987337
    key_type: IPV6
  }
  start_time: 1388563200987337
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200987337
}
flow_info {
  key {
    src_address: "p\202q\266j\035\024\242N\362}{!r\204\251"
    dst_address: "\307\334\327\251\211\302\006\365\305L0\240\363\005\241$"
    src_port: 39353
    dst_port: 33279
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563201045298
    key_type: IPV6
  }
  start_time: 1388563201045298
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 200
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563201045298
}
flow_info {
  key {
    src_address: "f\217\013\347nj\250\253\252d]\276I\263\274\267"
    dst_address: "\341\2030\234\027\265\'\024\376\326\200<>\005\367{"
    src_port: 1070
    dst_port: 9795
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563201103259
    key_type: IPV6
  }
  start_time: 1388563201103259
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563201103259
}
flow_info {
  key {
    src_address: "\262\256\2503\262\002\322\273e\305\3474^6`\004"
    dst_address: "\330\272!\320\362p\264\006\020w3\246\177\212\206N"
    src_port: 5060
    dst_port: 9802
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563201161220
    key_type: IPV6
  }
  start_time: 1388563201161220
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 220
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563201161220
}
flow_info {
  key {
    src_address: "p\202q\266j\035\024\242N\362}{!r\204\251"
    dst_address: "\217\356\231\256\212\211=GU\324\245\261$\235g/"
    src_port: 61501
    dst_port: 39060
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563201219181
    key_type: IPV6
  }
  start_time: 1388563201219181
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563201219181
}
flow_info {
  key {
    src_address: "f\217\013\347nj\250\253\252d]\276I\263\274\267"
    dst_address: "\262\256\2503\262\002\322\273e\305\3474^6`\004"
    src_port: 40894
    dst_port: 2463
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563201277142
    key_type: IPV6
  }
  start_time: 1388563201277142
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 240
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563201277142
}
flow_info {
  key {
    src_address: "\262\256\2503\262\002\322\273e\305\3474^6`\004"
    dst_address: "\307\334\327\251\211\302\006\365\305L0\240\363\005\241$"
    src_port: 51344
    dst_port: 11217
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563201335103
    key_type: IPV6
  }
  start_time: 1388563201335103
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563201335103
}
flow_info {
  key {
    src_address: "p\202q\266j\035\024\242N\362}{!r\204\251"
    dst_address: "\341\2030\234\027\265\'\024\376\326\200<>\005\367{"
    src_port: 17217
    dst_port: 61158
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563201393064
    key_type: IPV6
  }
  start_time: 1388563201393064
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 260
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563201393064
}
flow_info {
  key {
    src_address: "f\217\013\347nj\250\253\252d]\276I\263\274\267"
    dst_address: "\330\272!\320\362p\264\006\020w3\246\177\212\206N"
    src_port: 39353
    dst_port: 37778
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563201394064
    key_type: IPV6
  }
  start_time: 1388563201394064
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563201394064
}
flow_info {
  key {
    src_address: "\262\256\2503\262\002\322\273e\305\3474^6`\004"
    dst_address: "\217\356\231\256\212\211=GU\324\245\261$\235g/"
    src_port: 1070
    dst_port: 4921
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563201452025
    key_type: IPV6
  }
  start_time: 1388563201452025
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 280
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563201452025
}
flow_info {
  key {
    src_address: "p\202q\266j\035\024\242N\362}{!r\204\251"
    dst_address: "\262\256\2503\262\002\322\273e\305\3474^6`\004"
    src_port: 5060
    dst_port: 23104
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563201509986
    key_type: IPV6
  }
  start_time: 1388563201509986
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563201509986
}
flow_info {
  key {
    src_address: "f\217\013\347nj\250\253\252d]\276I\263\274\267"
    dst_address: "\307\334\327\251\211\302\006\365\305L0\240\363\005\241$"
    src_port: 61501
    dst_port: 63752
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563201567947
    key_type: IPV6
  }
  start_time: 1388563201567947
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 300
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563201567947
}
flow_info {
  key {
    src_address: "\262\256\2503\262\002\322\273e\305\3474^6`\004"
    dst_address: "\341\2030\234\027\265\'\024\376\326\200<>\005\367{"
    src_port: 40894
    dst_port: 33279
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563201625908
    key_type: IPV6
  }
  start_time: 1388563201625908
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563201625908
}
flow_info {
  key {
    src_address: "p\202q\266j\035\024\242N\362}{!r\204\251"
    dst_address: "\330\272!\320\362p\264\006\020w3\246\177\212\206N"
    src_port: 51344
    dst_port: 9795
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563201683869
    key_type: IPV6
  }
  start_time: 1388563201683869
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 320
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563201683869
}
flow_info {
  key {
    src_address: "f\217\013\347nj\250\253\252d]\276I\263\274\267"
    dst_address: "\217\356\231\256\212\211=GU\324\245\261$\235g/"
    src_port: 17217
    dst_port: 9802
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563201741830
    key_type: IPV6
  }
  start_time: 1388563201741830
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563201741830
}
flow_info {
  key {
    src_address: "\262\256\2503\262\002\322\273e\305\3474^6`\004"
    dst_address: "\262\256\2503\262\002\322\273e\305\3474^6`\004"
    src_port: 39353
    dst_port: 39060
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563201799791
    key_type: IPV6
  }
  start_time: 1388563201799791
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 340
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563201799791
}
flow_info {
  key {
    src_address: "p\202q\266j\035\024\242N\362}{!r\204\251"
    dst_address: "\307\334\327\251\211\302\006\365\305L0\240\363\005\241$"
    src_port: 1070
    dst_port: 2463
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563201857752
    key_type: IPV6
  }
  start_time: 1388563201857752
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563201857752
}
flow_info {
  key {
    src_address: "f\217\013\347nj\250\253\252d]\276I\263\274\267"
    dst_address: "\341\2030\234\027\265\'\024\376\326\200<>\005\367{"
    src_port: 5060
    dst_port: 11217
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563201858752
    key_type: IPV6
  }
  start_time: 1388563201858752
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 360
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563201858752
}
flow_info {
  key {
    src_address: "\262\256\2503\262\002\322\273e\305\3474^6`\004"
    dst_address: "\330\272!\320\362p\264\006\020w3\246\177\212\206N"
    src_port: 61501
    dst_port: 61158
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563201916713
    key_type: IPV6
  }
  start_time: 1388563201916713
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563201916713
}
flow_info {
  key {
    src_address: "p\202q\266j\035\024\242N\362}{!r\204\251"
    dst_address: "\217\356\231\256\212\211=GU\324\245\261$\235g/"
    src_port: 40894
    dst_port: 37778
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563201974674
    key_type: IPV6
  }
  start_time: 1388563201974674
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 380
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563201974674
}
flow_info {
  key {
    src_address: "f\217\013\347nj\250\253\252d]\276I\263\274\267"
    dst_address: "\262\256\2503\262\002\322\273e\305\3474^6`\004"
    src_port: 51344
    dst_port: 4921
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563202032635
    key_type: IPV6
  }
  start_time: 1388563202032635
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563202032635
}
flow_info {
  key {
    src_address: "\262\256\2503\262\002\322\273e\305\3474^6`\004"
    dst_address: "\307\334\327\251\211\302\006\365\305L0\240\363\005\241$"
    src_port: 17217
    dst_port: 23104
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563202090596
    key_type: IPV6
  }
  start_time: 1388563202090596
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 400
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563202090596
}
flow_info {
  key {
    src_address: "p\202q\266j\035\024\242N\362}{!r\204\251"
    dst_address: "\341\2030\234\027\265\'\024\376\326\200<>\005\367{"
    src_port: 39353
    dst_port: 63752
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563202148557
    key_type: IPV6
  }
  start_time: 1388563202148557
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563202148557
}
flow_info {
  key {
    src_address: "f\217\013\347nj\250\253\252d]\276I\263\274\267"
    dst_address: "\330\272!\320\362p\264\006\020w3\246\177\212\206N"
    src_port: 1070
    dst_port: 33279
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563202206518
    key_type: IPV6
  }
  start_time: 1388563202206518
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 420
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563202206518
}
flow_info {
  key {
    src_address: "\262\256\2503\262\002\322\273e\305\3474^6`\004"
    dst_address: "\217\356\231\256\212\211=GU\324\245\261$\235g/"
    src_port: 5060
    dst_port: 9795
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563202264479
    key_type: IPV6
  }
  start_time: 1388563202264479
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563202264479
}
flow_info {
  key {
    src_address: "p\202q\266j\035\024\242N\362}{!r\204\251"
    dst_address: "\262\256\2503\262\002\322\273e\305\3474^6`\004"
    src_port: 61501
    dst_port: 9802
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563202322440
    key_type: IPV6
  }
  start_time: 1388563202322440
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 440
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563202322440
}
